### ISSUE TITLE
1.4.0-beta.0 doesn't have some features and files from 1.4.0-alpha.1

### ISSUE THREAD

**Issue by aweber1** [callstack/linaria#523]
## Description

I may be missing something obvious, but after browsing the repo on GitHub and cloning the repo locally, it appears as though some of the features listed in the release notes for `1.4.0-beta.0` are not present.

In particular, the following PRs don't appear to be merged into the `1.4.0-beta.0` tag:
#430
#353 

There may be others missing, but those are the two I'm noticing for certain.

I see the files for those PRs in the `1.4.0-alpha.X` tags, but not in the beta tag. Also seeing the same when browsing the released npm package on local file system and at unpkg.

I noticed this issue with `1.4.0-beta.0` as I'm encountering a similar (but not same) error to #428 when trying to use the `linaria/babel` preset with TypeScript.

**Comment by Anber** [comment]
#353 is missed because everything about the new extractor has been moved to another PR (https://github.com/callstack/linaria/pull/499). It will be merged soon but as an experimental optional behaviour. I'll let you know when that part will be merged and documented.

Could you please describe an error which you encountered?

**Comment by aweber1** [comment]
With `1.4.0-beta.0` I was receiving the same error message mentioned in #428: 
`Error: [BABEL] unknown: Configuration contains string/RegExp pattern, but no filename was passed to Babel`

A fix for #428 was released in `shaker.ts` in `1.4.0-alpha.1`, however, `1.4.0-beta.0` doesn't have a `shaker.ts` file (or the files surrounding it), so I was making the conclusion that the beta release didn't have all the alpha release features/fixes. However, it sounds like perhaps this was known/expected from a maintainer standpoint.

I suspect the error was originating in this line from the beta tag source:
https://github.com/callstack/linaria/blob/0f314a51efa3929cf90016db757577e737c811a6/src/babel/evaluators/extractor/index.ts#L55

But didn't have time to fully investigate / troubleshoot.

Switching my library to use `1.4.0-alpha.1` fixed the problem for me.

**Comment by aweber1** [comment]
Feel free to close this issue if there aren't any action items or if the upcoming PR #499 (or an upcoming release) will be comprehensive. Understandably, `beta` and `alpha` are not "final" :) 

I opened this issue more for awareness of the discrepancies between `alpha` and `beta` in the event other people run into similar situations.

**Comment by Anber** [comment]
Ooops. You are right. I've missed a filename in `evaluators/extractor/index.ts`. Thank you!

**Comment by Anber** [comment]
Hi @aweber1 

I've merged #499  and released a `beta.1`.

You can try to use the new evaluation strategy by adding `rules` option to your linaria config file:
```
const shaker = require('linaria/lib/babel/evaluators/shaker').default;

module.exports = {
    rules: [
        {
            action: shaker,
        },
        {
            test: /\/node_modules\//,
            action: 'ignore',
        },
    ],
};
```

Please notice that "rule" option is experimental and may be changed in the near future.
